mapreduce-pipeline/generate_plot/compile_plot.py

```diff
@@ -63,9 +63,9 @@ def generate_plot(output_file, experiments):
 
     # Kubeflow doesn't render the .png file nicely in the browser
     # So need this to turn it into an ascii html page.
-    import base64
-    encoded = base64.b64encode(open(output_file, "rb").read()).decode('ascii')
-    with open("out.html", "w") as f:
-        f.write(f'<img src="data:image/png;base64,{encoded}" />')
+    # import base64
+    # encoded = base64.b64encode(open(output_file, "rb").read()).decode('ascii')
+    # with open("out.html", "w") as f:
+    #     f.write(f'<img src="data:image/png;base64,{encoded}" />')
```

Comment:
does this example actually produce the visualization, or is that not working?  If not working, we should make sure the user knows that

Reply:
Able to view visualization now.